### DISCUSSION TITLE
特定服务的调度问题

### DISCUSSION THREAD

**Discussion by cloudfreexiao** [cloudwu/skynet#1282] [General]
在游戏里 战斗 场景 这些是 热点 业务 导致 其他 依赖这些 服务受到 拖累
skynet 里 能否 让这些业务 由 业务上层设置 调度权重 或者 独立 线程跑

**Comment by cloudwu** [comment]
不能。但是，是否真的检测到重要服务被饿死？还是只是设想？

**Comment by cloudfreexiao** [comment]
在 压测 阶段 有特定几个服务 由于 业务原因 负载比较高 会出现 消息堆积的问题
拆分的过于细 又觉得 对 业务开发 带来 复杂度
就是想 能不能 提供些 变通的办法 在这两者 求点平衡

**Comment by cloudwu** [comment]
有什么证据证明消息堆积是因为服务被饿死，也就是说 cpu 算力足够，但是没有处理那几个堆积消息的服务？

我认为消息堆积并不能通过提高优先级来解决。而且现在如果被头几个线程抽到，work thread 本来就会一次处理一批消息。
